packages/runtime/src/helpers/logger/winston.ts

```diff
@@ -0,0 +1,11 @@
+import winston from 'winston'
+
+// Define log level from environment variable so we can change it if needed
+const logLevel = process.env.LOG_LEVEL ?? 'info'
```

Comment:
This will allow us to only display `logger.info` and higher levels (`.warn`, `.error`) into the user-facing logs.

We can use the environment variable with customers or support so we can use `logger.debug` or similar in the code.